Upstream: sent https://github.com/kislyuk/argcomplete/pull/266

commit 7a8305510bfbbaeed4a83be93d7e64ce0d8439a5
Author: Alexander Kapshuna <kapsh@kap.sh>
Date:   Thu Aug 22 13:15:05 2019 +0300

    Skip tests for fish and tcsh when they are not installed

diff --git a/test/test.py b/test/test.py
index 8045f74..aef4835 100755
--- a/test/test.py
+++ b/test/test.py
@@ -1215,6 +1215,7 @@ class TestBashGlobal(TestBash):
         self._test_console_script(package=True, wheel=True)


+@unittest.skipIf(pexpect.which('tcsh') is None, 'tcsh is not installed')
 class TestTcsh(_TestSh, unittest.TestCase):
     expected_failures = [
         'test_unquoted_space',
@@ -1243,6 +1244,7 @@ class TestTcsh(_TestSh, unittest.TestCase):
             self.sh.run_command('')


+@unittest.skipIf(pexpect.which('fish') is None, 'fish is not installed')
 class TestFish(_TestSh, unittest.TestCase):
     expected_failures = [
         'test_parse_special_characters',
